CMakeLists.txt

```diff
@@ -188,7 +189,7 @@ ELSE()
   KOKKOSKERNELS_ADD_TEST_DIRECTORIES(test_common)
   KOKKOSKERNELS_ADD_TEST_DIRECTORIES(perf_test)
   KOKKOSKERNELS_ADD_TEST_DIRECTORIES(unit_test)
-  #KOKKOSKERNELS_ADD_EXAMPLE_DIRECTORIES(example)
+  KOKKOSKERNELS_ADD_EXAMPLE_DIRECTORIES(example)
```

Comment:
I would squash the two commits.

Reply:
I figured I would get comments on the CMake side at least so I felt like squashing was going to be an inevitability before the PR merges ; )